model/src/main/java/com/basistech/rosette/dm/Embeddings.java

```diff
@@ -0,0 +1,118 @@
+/*
+* Copyright 2016 Basis Technology Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.basistech.rosette.dm;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Embeddings for a text.
+ * A text can have embedding values for the entire text and/or one of more of the collections of
+ * items inside the text, such as the tokens or entities. The collections are identified by strings.
+ * Constants are provided for some common cases.
+ */
+public class Embeddings extends BaseAttribute {
+    //* The embedding values for the entire text.
+    public static final String TEXT = "text";
```

Comment:
So that's purely for *our* convenience, whereas an enum would be for the convenience of the API users.  I'm not sure these will change often enough for this to matter.  Wasn't there some issue about json serialization?